CMakeLists.txt

```
@@ -678,7 +678,7 @@ if(USE_ASAN)
     string(APPEND CMAKE_LINKER_FLAGS_DEBUG " -fsanitize=address")
 endif()
 
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(aarch64|arm64)$")
```

Not needed here, as `clang` for M1 supports VST1 intrinsics
```suggestion
if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
```

---

you mean if we use the preferred compiler `clang` in MacOS (on M1), the source compile check is unnecessary? (in this case CMAKE_SYSTEM_PROCESSOR is `arm64` yes?)